Bottom: b0ecaac03a4cee3fc35723b2693d5f56a0be1c5b
Top:    dc558b29b58b817bb5cd9c51d9a945b62f4283dd
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-22 20:35:30 +0000

Refresh of hid-gtk-disable-gl-alpha-blend

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index dad2d61..ee0e7dc 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1083,8 +1083,8 @@ ghid_pinout_preview_expose (GtkWidget *widget,
   ghid_start_drawing (gport, widget);
   hidgl_start_render ();
 
-  glEnable (GL_BLEND);
-  glBlendFunc (GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
+//  glEnable (GL_BLEND);
+//  glBlendFunc (GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
 
   glViewport (0, 0, allocation.width, allocation.height);
